start-client/src/components/common/error/Boot.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+class Boot extends React.Component {
+  render = () => {
+    return (
+      <div className='control-error'>
+        <p className='title'>
+          <strong>
+            The version {this.props.value} of Spring Boot is outdated.
+          </strong>
+        </p>
+        <p>
+          You have to select a valid version if you want to generate or explore
+          the project.
+        </p>
+        <p>
+          <a
+            href='/#'
+            onClick={e => {
+              e.preventDefault()
+              this.props.selectLatest()
+            }}
+          >
+            > Select the latest stable version
```

Comment:
Could this include what the version is, so something like "Select 2.1.7 (the latest stable version)"?